packages/core/src/lib/markdown-it/patches/custom-component/htmlBlockRule.js

```diff
@@ -11,7 +11,7 @@ function initCustomComponentHtmlBlockRule(tagsToIgnore) {
    * Custom patch for the api to escape content in certain special tags.
    */
   const specialTagsRegex = Array.from(tagsToIgnore)
-  .concat(['include|script|pre|style|variable|site-nav|markdown|md'])
```

Comment:
👀 should we remove the later one instead? less regex runs

Reply:
Yep sure! I have shifted the `script`, `pre`, and `style` to the back of the array here and made a comment that they are for Vue, just for documentation :-)